### DISCUSSION TITLE
Version update stuck at 3.3.0

### DISCUSSION THREAD

**Discussion by MrBertie** [darlal/obsidian-switcher-plus#129] [Q&A]
Since about 3 months my plugin version is stuck at 3.3.0 although the current latest version is 3.3.5. The Obsidian plugin manager also shows 3.3.0 as the latest version number.
Anyone else noticed the same situation?

**Comment by darlal** [comment]
Plugin versions map to one or more Obsidian versions, so there has to be a plugin versions compatible with your Obsidian version. Most important thing:

- What Obsidian version are you running? 

You can check this in the Obsidian preferences "About" tab. If you're running a version less than `1.3.4` then plugin version `3.3.0` could make sense. In that case, you might want to upgrade to the latest Obsidian release.

If you're running the latest Obsidian version `>= 1.3.5`, then the latest Switcher++ version `3.3.5` should be compatible. In that case there's something else going on. I would recommend:

1. Check if you have any of the "plugin manager" type plugins installed, perhaps that is causing issues
2. Check if you have any of the "plugin auto updater" type plugins installed, same as above
3. Check the Obsidian developer console to see if any errors are being reported View->Toggle Developer Tools menu

Worst case, you can backup your switcher++ settings file, then uninstall the plugin and reinstall it. The settings file is located in `vault-root/.obsidian/plugins/darlal-switcher-plus/data.json`

Good luck, let me know how it goes.
